### ISSUE TITLE
generatePerson sometimes generates invalid id

### ISSUE THREAD

**Issue by Herdismaria** [HermannBjorgvin/Kennitala#35]
It looks like either the `generatePerson` function or `info` has some bug in it. If I use `generatePerson` to create a kennitala and then use `info` to parse it, `info` sometimes results in invalid kennitala.

```
it('should return a valid national id', () => {
      const nationalId = generatePerson(new Date())!
      const nationalIdInfo = info(nationalId)
      expect(nationalIdInfo?.valid).toEqual(true)
    })
```

Example of a kennitala created with generatePerson

```
  {
      kt: '0101210190',
      valid: false,
      type: 'invalid',
      birthday: Invalid Date,
      birthdayReadable: '',
      age: NaN
    }
```

This test is flaky. I'm using version 2.0.6.

**Comment by Herdismaria** [comment]
Also should the Date parameter be optional for `generatePerson`? it looks like it's required now :)

**Comment by HermannBjorgvin** [comment]
Thanks for the bug report @Herdismaria 

I think the tests might be failing because like the KT you generated shows it has a 7th-8th digits of `01` which would be illegal for persons since they start generating personal ID's at `20` and up.

Looks like the bug is residing in `./src/generation.ts` where `startingIncrement` should have a default value of `20`.

I'll push out a fix for after work today. Apologies for the inconvenience. If you want a quick fix you can add `20` as an optional parameter to your `generatePerson` function call.

```ts
const generatePerson = (
  date: Date,
  startingIncrement?: number
): string | undefined => {
  return generateKennitala(date, personDayDelta, startingIncrement);
};
```

**Comment by HermannBjorgvin** [comment]
> Also should the Date parameter be optional for `generatePerson`? it looks like it's required now :)

Interesting thought. So if the Date parameter is optional we could generate a random Date as a default parameter?

Would be a cool feature

**Comment by HermannBjorgvin** [comment]
Fixed in #36 

Thanks for creating the issue!

**Comment by HermannBjorgvin** [comment]
Fixed in V3.0.0 that was released just now. Thanks a lot for reporting this, I think it would have gone unnoticed for a while otherwise :)
